secp256k1-sys/src/lib.rs

```diff
@@ -517,11 +527,44 @@ impl core::hash::Hash for Keypair {
     }
 }
 
+pub struct XOnlySharedSecret(pub [u8; 32]);
+
+impl XOnlySharedSecret {
+    pub fn as_bytes(&self) -> &[u8] {
+        &self.0
+    }
+    pub fn as_mut_bytes(&mut self) -> &mut [u8] {
+        &mut self.0
+    }
+}
+
+impl_array_newtype!(XOnlySharedSecret, u8, 32);
+impl_raw_debug!(XOnlySharedSecret);
+
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct ElligatorSwift([u8; 64]);
+
+impl ElligatorSwift {
+    pub fn from_array(arr: [u8; 64]) -> Self {
+        ElligatorSwift(arr)
+    }
+    pub fn into_array(self) -> [u8; 64] {
```

Comment:
nit: This should be `to_array` because the array is `Copy`

ref: https://rust-lang.github.io/api-guidelines/naming.html (search for "into_")